### PR TITLE
Add more info on umbracoContentXMLStorage following on from #610

### PR DESCRIPTION
**Pull request by Jeavon**

### CODE REVIEW

**Comment by Jeavon** [comment]
@Shazwazza wanted to check you agreed with this bit https://github.com/umbraco/UmbracoDocs/pull/618/files#diff-d6929582651fe6d81bf18392f0d9c53cR71

File: Getting-Started/Setup/Server-Setup/azure-web-apps.md [hunk 1]
```diff
@@ -62,15 +62,17 @@ The `SyncTempEnvDirectoryFactory` enables Examine to sync indexes between the re
...
 * Due to the nature of Lucene files and IO latency, you should update all of your Indexers and Searchers in the `~/Config/ExamineSettings.config` file to have these two properties (see [here](http://issues.umbraco.org/issue/U4-7614) for more details): `useTempStorage="Sync"`
 
-### Umbraco XML cache file
+### Umbraco XML cache file and other TEMP files
 
 For a single Azure Web App instance you need to ensure that the Umbraco XML config file is stored on the local server (since Azure uses a shared file system). To do this you need to add a new app setting to web.config:
 
 For **Umbraco v7.7.3+**
 
+It is recommended that you upgrade your Azure Web App to support this setting to avoid locking issues with TEMP files during server migrations or slot swapping. See [U4-10503](http://issues.umbraco.org/issue/U4-10503) for more information on this.
```
Comment (dampee) [inline Getting-Started/Setup/Server-Setup/azure-web-apps.md]: what do you mean with "support your Azure Web App"?

Comment (Jeavon) [reply Getting-Started/Setup/Server-Setup/azure-web-apps.md]: Good point, how about “upgrade your Azure Web App hosted Umbraco”

Comment (Jeavon) [reply Getting-Started/Setup/Server-Setup/azure-web-apps.md]: I have rephrased the whole paragraph now

**Comment by Jeavon** [comment]
@nul800sebastiaan @Shazwazza before merging this, I wanted to check that you are agreeable to making v7.7.3 the recommended minimum Umbraco version for Azure Web Apps? https://github.com/umbraco/UmbracoDocs/pull/618/files#diff-d6929582651fe6d81bf18392f0d9c53cR71

**Pull request merged**